### PR TITLE
Feature/terms and conditions

### PR DESCRIPTION
**Pull request by just-paja**
Contains PR #74 .

### CODE REVIEW

**Comment by PetrDlouhy** [comment]
Will be all rules, terms and conditions represented as 1) one text or 2) multiple texts?

In case 1) this should be bound to `Year` as OneToOne to ensure, that there is only one such object.
In case 2) we would need either `slug` field or `type` field with choices `rule`, `term` or `condition`.

Maybe we should move this to the application `api_textual` to achieve better code division.

**Comment by just-paja** [comment]
@PetrDlouhy, there will be multiple Terms & Conditions per year, because it is necessary to keep older versions stored.

**Comment by PetrDlouhy** [comment]
@just-paja You didn't answer my question, just raised more confusion.

How would the backend decide, which `Rules` record should be retrieved? There is no identificator in the model.

If there should be stored more versions of the rules, how would I tell, which one is the current?

**Comment by just-paja** [comment]
@PetrDlouhy, the latest.

**Comment by PetrDlouhy** [comment]
@just-paja The latest by `created_at`? What if we need to recover the older version of the `Rules`?
Wouldn't be better to add some field, which the Admin could modify?

**Comment by just-paja** [comment]
@PetrDlouhy, if we need to recover older version, we just copy and reuse the old version with newer date, so that participants can see the changes.

File: api/models/rules.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class Rules(Base):
+    """Stores rules, terms and conditions for year."""
+
+    year = models.ForeignKey(
+        Year,
+        verbose_name=_("Year"),
+    )
+    text = models.TextField(
```
Comment (PetrDlouhy) [inline api/models/rules.py]: Maybe we could use `MarkdownField` for this as in PR #82.

Comment (just-paja) [inline api/models/rules.py]: Markdown field is not required because markdown is admin-human readable.

Comment (PetrDlouhy) [inline api/models/rules.py]: Not required, but it's nice to have markdown editor to help you remember the Markdown features. And it is very easy to use.

Comment (just-paja) [inline api/models/rules.py]: You can read manual to remember the Markdown features. It's easy to use, but extra responsibility to maintain.

Point of having it like this is to **limit** the admin from being distracted by extra features while he should be creating **text content**. We can also have TinyMCE with all features and all plugins for headings, tables, forms, iframes, flash movies and gif editors, but why do we need that? We don't. Should we pull in dependencies for something we don't need? We shouldn't.

Comment (PetrDlouhy) [inline api/models/rules.py]: I don't agree. The preview, checking and other features would be worth the effort for me.

But I already approved and you are free to merge. You will be the admin anyway.

Comment (just-paja) [inline api/models/rules.py]: @PetrDlouhy, I don't know why are we going back to this again and again. We shouldn't develop features nobody asked for. It's a waste of time.

Comment (PetrDlouhy) [inline api/models/rules.py]: Proč potom komunikujeme v angličtině? Jak to pomůže uživatelům?

**Pull request merged**